embedded-wallets/authentication/login/oauth/social-providers/apple.mdx

```diff
@@ -1,5 +1,5 @@
 ---
-title: Apple Social Login with Magic
```

Comment:
What about for this (and the other pages) `Apple OAuth`. Detailed titles helps the search functionality within Mintlify.